ethpm/package.py

```diff
@@ -30,6 +28,11 @@ def __init__(self, lockfile):
             validate(package_data, schema_data)
         except jsonValidationError:
             raise ValidationError
+
+        if 'build_dependencies' in package_data.keys():
+            if package_data['build_dependencies']:
+                raise NotImplementedError
```

Comment:
@pipermerriam nested `if`s smell funny to me. Is this how you would do it?

Reply:
I'd do something like:

```python
if package_data.get('build_dependencies'):
    ...
```

The `.get(...)` method on dictionaries return `None` by default if the key isn't present so this should work for all scenarios that need to be handled.

Reply:
Yeah, that's much better. Thanks!